### PR TITLE
update the width to match the parent.

### LINKED ISSUE #2057: Image shows different size bug
**Describe the bug**
Every time you go back and forth to that page that has an image, it will show you different image size.

**Component**
SDC library

**To Reproduce**
1. Create paginated questionnaire
2. Have this itemMedia extension to a questionnaire.item
```
{
  "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-itemMedia",
  "valueAttachment": {
    "contentType": "image/png",
    "data": "base64 image",
    "title": "Discussion"
  }
}
```
3. Every time you go back and forth to that page that has an image, it will show different image size

**Expected behavior**
Image size should stay consistent

**Screenshots**
 

**Smartphone (please complete the following information):**
 - Device: [e.g. Pixel4a emulator]
 - Android version: [e.g. Settings -> About phone -> Android version]
 - Build number: [e.g. Settings -> About phone -> Build number]

**Additional context**
- Sample Questionnaire https://drive.google.com/file/d/18jGWg5AxCHra488OC2kMo2Q-_1FWa-xR/view?usp=sharing

**Would you like to work on the issue?**
N/A

### PR DESCRIPTION
**Pull request by santosh-pingle**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #2057 

**Description**
Update the imageview width to match the parent.

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: Bug fix 

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Comment by santosh-pingle** [comment]
Before Fix 



https://github.com/google/android-fhir/assets/86107848/f610aca3-006f-4992-8094-4f7796078fbf

After fix : 


https://github.com/google/android-fhir/assets/86107848/38cde211-fd81-4052-accd-df8c3019a097

**Review by FikriMilano — APPROVED** [review]
@santosh-pingle looks good to me, thanks :D

File: datacapture/src/main/res/layout/media_view.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <ImageView
         style="?attr/questionnaireMediaImageStyle"
         android:id="@+id/image_attachment"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
```
Comment (jingtang10) [inline datacapture/src/main/res/layout/media_view.xml:12]: why does this change fix the problem?

File: datacapture/src/main/res/layout/media_view.xml
[the remark this answers is not in the document]
Comment (santosh-pingle) [reply datacapture/src/main/res/layout/media_view.xml:1]: Actually, media_view.xml is inflated in MediaView.
LinearLayout is root layout to position the imageView in the container.
If the imageView has a width and height set to `wrap-content`, then the positioning of the imageView gets disturbed, and positioning attributes on the imageView do not work as expected. The imageView is always displayed to the left.

File: datacapture/src/main/res/layout/media_view.xml (hunk 1 above)
Comment (santosh-pingle) [reply datacapture/src/main/res/layout/media_view.xml:12]: The ImageView has maxWidth and/or maxHeight values set to 200dp. After setting the width to match_parent, the ImageView's width is adjusted to match_parent. However, due to the smaller maxWidth and/or maxHeight values, the image inside the ImageView gets scaled down to fit within these constraints while preserving its aspect ratio

**Review by jingtang10 — APPROVED** [review]

**Pull request merged**